include/mbgl/style/function.hpp

```diff
@@ -15,19 +23,26 @@ class Function {
     explicit Function(Stops stops_, float base_)
         : base(base_), stops(std::move(stops_)) {}
 
+    explicit Function(Stops stops_, float base_, ColorSpace colorSpace_)
```

Comment:
Combine with the previous constructor using a default parameter (`ColorSpace colorSpace_ = ColorSpace::RGB`).